roles/storage-cinder/defaults/main.yml

```diff
@@ -0,0 +1,69 @@
+---
+action: provision
+namespace: openstack
+service_account: cinder
+privileged_service_account: cinder-privileged
+
+storage_cinder_template_dir: "{{ role_path }}/templates"
+
+mariadb_root_user: root
+mariadb_root_password: weakpassword
+
+cinder_user: cinder
+cinder_password: cinderpassword
+
+rabbimq_user: guest
+rabbitmq_password: rabbitmqpassword
+
+auth_strategy: noauth
+
+cinder_config:
+  DEFAULT:
+    public_bind_host: "0.0.0.0"
+
+role_name: cinder
+
+infra_node_label: mariadb
+
+# Backends
+# Array to remember enabled backends. 
+enabled_backends: []
+
+iscsi:
+    enabled: false
+
+#
+# Xtremio
+xtremio:
+    enabled: false
```

Comment:
Rename `enabled` to `prefix_enabled`, please

Reply:
Same as above. The 'enabled' var is basically at <cinder-backend>.enabled. So, I'm hesitant to change it.

Reply:
I am ok with both ways ...
```yaml
group:
    var_name: XYZ
```
either
```yaml
group_var_name: XYZ
```

On other hand I case see on line [62](https://github.com/kubevirt/kubevirt-ansible/pull/169/files/8ad35d4f677d964b7c0aa7fa2224800e55a83cec#diff-3f8e79ab052fe339f8e49f186f68f56dR62)
```yaml
netapp:
    netapp_storage_family: ontap_cluster
```
which seems to me as a duplication of group name in nested variable name.

Reply:
Those Cinder backend variables came from TripleO Cinder and Puppet Cinder projects. The 'enabled' variable is original to this role. Some backends use prefixed vars, others don't. Its not consistent.

Reply:
The 'enabled' var is referenced in many files within the role and changing it just for the sake of it makes no sense to me. Thanks.

Reply:
I prefer we keep it the way Kiran has it now.